src/dsp_tools/resources/schema/data.xsd

```diff
@@ -225,9 +247,14 @@
 
     <!-- textprop_type -->
     <xs:complexType name="textprop_type">
-        <xs:sequence>
-            <xs:element name="text" type="text_type" minOccurs="1" maxOccurs="unbounded"/>
-        </xs:sequence>
+        <xs:choice minOccurs="1" maxOccurs="1">
```

Comment:
For some reason this choice does not work, it now only accepts xml encoding, if I switch the order and utf8 is on top it then only accepts utf8.

I have also tried to shift the cardinality around, this also does not work and weirdly I still get the same error message. 
@jnussbaum @BalduinLandolt  do you have any ideas?

Reply:
I haven't worked with XML schema in a long time, sorry... Some thoughts:
- for now, simply test that in code instead of the XML, but in the "validate XML" phase of the code?
- maybe this is an indication of another need to change the structure of our XML? if simply have `<SimpleText>` vs. `<FormattedText>`, then things would get a lot easier to validate, right?

Reply:
From what I have read, a big problem is having different types with the same name, here "text" so yes if we could have an either or it should be better, according to my understanding.

Reply:
what do you mean by:
for now, simply test that in code instead of the XML, but in the "validate XML" phase of the code?

Reply:
> what do you mean by: "for now, simply test that in code instead of the XML, but in the "validate XML" phase of the code"

Either we have, or we could have (not sure how it is exactly) some code that is executed *after* reading the XML (checking that the file exists, etc.) and *before* parsing the XML to domain objects (our beloved `XMLResource` etc.) that does validation on the XML file programmatically, beyond what we can do with the schema. [I'm pretty sure this kind of code existed at one point - not sure though, if it still does.]  
This could be a place to do validation programmatically, which is annoying to write in the schema.  
And checking this constraint would be pretty trivial in python, right?

> But changing the tag names would be a breaking change ...

Sure, this can't happen in the present PR. But it's something we want to do anyways, so this point is something to take note of for when we do this change, so we can be sure, the new XML format will make this kind of validation not too hard.

Reply:
ok, sounds good, thanks. I will change the hasComment to mandatory xml encoding, since that is an easy fix. And open another PR for the validation.

Reply:
Stackoverflow helped, it is not possible in XML-Schema 1.0, in 1.1 it is possible, but switching from one to the other is not so simple and the validation engine may not accept it. So ultimately it seems easier that we do it in code.